Link_sinFade_Library.ino

```diff
@@ -240,6 +240,7 @@ void HSVwriteToLEDs(float someHSV[], int someZone) {  // <<< Includes whiteBalan
   for (byte led = 0; led < 3; led++) {
     PWM.set(LunitZones[someZone][led], RGB_from_hsv[led]);  // Display color step.
   }
+  PWM.commit(); // for spi/i2c devices, try to batch more updates at once
```

Comment:
Well, I needed this for the more general pattern, things like the TLC59711 (i2c or spi boards, or just smarter protocols....) will batch up the changes (each `set()` is saved), and the `write` it.

So, for native pins, commit() does nothing (see previous pull request #2 ). And now, changing the `#include` and the line after it is the only change necessary. This line doesn't have to change.